source/code-generation/query-converter/bulk-conversion.txt

```diff
@@ -0,0 +1,100 @@
+.. _rm-bulk-conversion:
+
+===============
+Bulk Conversion
+===============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can select and convert one or more queries, stored procedures, 
+triggers, and views in bulk by enabling the multi-select toggle. 
+Bulk conversion creates a job that queues and executes each query 
+conversion individually from a single request.
+
+Copy and paste your queries into the query converter individually and 
+then bulk convert a selection to update them to work with MongoDB and
+your migrated schema.
+
+About this Task
+---------------
+
+.. include:: /includes/fact-query-converter-generic.rst
+
+- .. include:: /includes/fact-query-converter-disclaimer.rst
+
+- .. include:: /includes/fact-view-query-conversion-history.rst
+
+Steps
+-----
+
+.. procedure::
+   :style:  normal
+
+   .. step:: Navigate to the query converter pane
+
+      From the :guilabel:`Code Generation` tab, click the 
+      :guilabel:`Query Converter` pane.
+
+   .. step:: Open the SQL query converter view
+
+      - If it is your first time using the query converter in your 
+        project, click :guilabel:`Paste SQL Query`.
+      - If you have previously converted SQL code in your project, click the
+        :guilabel:`+ ADD` button on the left-hand pane next to 
+        :guilabel:`Queries`.
+
+   .. step:: Enable multi-selection
+
+      Click the :guilabel:`Enable multi-selection` toggle to show the query 
+      selection pane.
+
+   .. step:: Select your conversions
```

Comment:
The "filter by status" feature is especially useful here so it would be nice to reference it.

Reply:
Added a link, this is currently broken due to separate PRs but will resolve in the feature branch. We'll track the error in our build log.